src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -6472,28 +6475,12 @@ namespace HVACVariableRefrigerantFlow {
 		// Determine operating mode based on VRF type and thermostat control selection
 		{ auto const SELECT_CASE_var( VRF( VRFCond ).ThermostatPriority );
 		if ( SELECT_CASE_var == ThermostatOffsetPriority ) {
-			TUIndex = VRF( VRFCond ).MasterZoneTUIndex;
-			if ( VRFTU( TUIndex ).FanOpModeSchedPtr > 0 ) {
-				if ( GetCurrentScheduleValue( VRFTU( TUIndex ).FanOpModeSchedPtr ) == 0.0 ) {
-					FanOpMode = CycFanCycCoil;
-				} else {
-					FanOpMode = ContFanCycCoil;
-				}
-			}
 			if ( MaxDeltaT( VRFCond ) > std::abs( MinDeltaT( VRFCond ) ) && MaxDeltaT( VRFCond ) > 0.0 ) {
 				HeatingLoad( VRFCond ) = false;
 				CoolingLoad( VRFCond ) = true;
 			} else if ( MaxDeltaT( VRFCond ) < std::abs( MinDeltaT( VRFCond ) ) && MinDeltaT( VRFCond ) < 0.0 ) {
 				HeatingLoad( VRFCond ) = true;
 				CoolingLoad( VRFCond ) = false;
-				// assuming if constant fan mode then previous operating mode will be needed
-				// could actually run the master zone TU to see if overshoot occurs, but seems overkill
-			} else if ( FanOpMode == ContFanCycCoil && LastModeCooling( VRFCond ) ) {
-				HeatingLoad( VRFCond ) = false;
-				CoolingLoad( VRFCond ) = true;
-			} else if ( FanOpMode == ContFanCycCoil && LastModeHeating( VRFCond ) ) {
-				HeatingLoad( VRFCond ) = true;
-				CoolingLoad( VRFCond ) = false;
```

Comment:
OK, now here. I can see the intent here with this code, albeit wrong as implemented. These ELSE IFs looked to the fan operating mode schedule to enable cooling/heating if MaxDeltaT/MinDeltaT were 0. For example, if the previous time step met the load, then it is possible that the zone temp = SP. If VRF TUs use constant fan mode, that would pose a load on the zone and this code tried to set operating mode based on the last operating mode. The problem here was that it looked to the master thermostat location to make that determination. That was incorrect in 1) no master thermostat is needed for this control, and 2) the load due to constant fan would be zone specific. IF this were to be done, it would need to be done up at line 6320-6330 ish, where we loop through all the terminal units. It's just that I am not sure what that logic would look like. For example, if a zone was at the SP, and used constant fan, and last mode was cooling, what would I do? Add max(MaxDeltaT(), 1) to the cooling var MaxDeltaT. That was the only thing I could think of doing last night IF I were to implement something here. Bottom line is that ThermostatOffsetPriority is meant to set the cooling/heating operating mode based on the worst case zone deviation from SP. I thought that just reverting back to that methodology was sufficient for now.